README.md

```diff
@@ -105,6 +302,9 @@ Create a PowerDNS recursor configuration named 'my-recursor' in your wrapper coo
       variables(client-tcp-timeout: '20', loglevel: '5', network-timeout: '2000')
     end
 
+## Contributing
+
+We are happy to accept contributions to this cookbook in form of bug fixes, new backends, init services, or platform support. We believe that barriers to contributing to this cookbook has been lowered since the release of the 3.0, resource based version. Please use the normal PR workflow for contributing. We will favour PRs with tests for the changes, we use both Chefspec and Inspec testing framewowrks.
```

Comment:
To better comply with new Supermarket linting, I'd extract this into a CONTRIBUTING.md document and note it here. I'd also make a testing section note and put that in a TESTING.md document so people know how they can test this cookbook themselves.